extensions/rich_text_components/Skillreview/directives/skillreview.component.html

```
@@ -3,7 +3,7 @@
   and opening or closing tag. This would make unnecessary spaces
   appear at the beginning or end of the link text.
 -->
-<a href="#"
+<a href="javascript:void(0)"
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)